lldb-eval/eval_test.cc

```diff
@@ -1696,3 +1696,61 @@ TEST_F(EvalTest, TestBuiltinFunction_Log2) {
       Eval("::ns::dummy(1)"),
       IsError("function '::ns::dummy' is not a supported builtin intrinsic"));
 }
+
+TEST_F(EvalTest, TestPrefixIncDec) {
```

Comment:
I noticed that LLDB allows `++(++$x)`, but `lldb-eval` results with `expression is not assignable`.

Reply:
It is allowed in C++ (and we can actually implement it too), but MSVC debugger doesn't allow such side-effects:
```
Natvis: C:\Users\werat\src\cpp\Dummy\NatvisFile.natvis(19,17): Error: Side effects are not supported in this context.
    Error while evaluating '++(++index)' in the context of type 'Dummy.exe!Foo'.
```

So for now we can get away without it too. I've added a comment about this.

Reply:
Thanks for the comment and clarification.